index.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable prefer-arrow-callback */
```

Comment:
Do we need this?

Reply:
Whoops, my auto eslint kept changing my functions into arrow functions so I added that to stop them.

Reply:
I'm fine with arrows all over

Reply:
But not eslint ignores

Reply:
I'm not sure what minimum version of node you want to support but arrow functions were added a bit later.

I've resolved this in the latest commit.

Reply:
Node 4 is the lowest :)